include/ocpp/v2/charge_point_callbacks.hpp

```diff
@@ -161,6 +161,9 @@ struct Callbacks {
                                      std::optional<std::string> currency_code)>>
         set_running_cost_callback;
 
+    /// \brief Callback function is called when tariff message is set.
```

Comment:
```suggestion
    /// \brief Callback function is called when a tariff message from the CSMS message has been received.
```

Reply:
I think it would help to give a little bit more context here and to describe whats expected from the callback.